vitals/BatteryCheck.java

```diff
@@ -0,0 +1,50 @@
+package vitals;
+
+public class BatteryCheck {
+	public static float UPPER_LIMIT_TEMPERATURE = 45;
+	public static float LOWER_LIMIT_TEMPERATURE = 0;
+	public static float UPPER_LIMIT_SOC = 80;
+	public static float LOWER_LIMIT_SOC = 20;
+	public static float MAX_CHARGE_RATE = 0.8f;
+	
+	float temperature;
+	float soc;
+	float chargeRate;
+
+	public BatteryCheck(float temperature, float soc, float chargeRate) {
+		this.temperature = temperature;
+		this.soc = soc;
+		this.chargeRate = chargeRate;
+	}
+	public boolean isBatteryOk() {
```

Comment:
isBatteryOk - What is the responsibility of this function , How to open this function for future changes ...